### ISSUE TITLE
A11y_AzureDataStudio_AddNewController_AI4D: Incorrect name is provided for  "Authentication Type" combo box.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7606]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1,#WCAG4.1.2;#DesktopApp;#NVDA;#JAWS;#AI4D;#SQL Azure Data Studio;#Win32
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : AI4D

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Connect to the servers with the required credentials.
3. Navigate to the "SQL Server Big Data Cluster" and expand it.
4. Navigate to the "SQL Server Big Data Cluster controller" and click on it.
5. Navigate till "Authentication Type " combo box using tab.
6. Check the control name provided for "Authentication Type" combo box.

**Actual:**
"Basic Combo box" name is provided for "Authentication Type" combo box.

**Expected:**
Correct name should be provided for the "Authentication Type" combo box.

**User Impact:**
Screen reader user will get incorrect information regarding the control name. 

**Attachment for Reference:**
![7606_A11y_AzureDataStudio_AddNewController_AI4D](https://user-images.githubusercontent.com/54656883/66485448-780c3180-eac6-11e9-8a39-8e815a65c749.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
This should be fixed in the latest insiders build 

![image](https://user-images.githubusercontent.com/28519865/68811298-2a369c00-0625-11ea-9997-35c5c8460396.png)

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7606_Closed](https://user-images.githubusercontent.com/54656883/72725212-31d96a80-3bab-11ea-9378-db46284e4c5e.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
